Makefile

```diff
@@ -2,6 +2,7 @@ GOOS ?= $(shell go env GOOS)
 GOARCH ?= $(shell go env GOARCH)
 SOURCES := $(shell find . -type f  -name '*.go')
 
+LDFLAGS='$(shell hack/version.sh)'
```

Comment:
What's the difference between this `LDFLAGS` with the one below?
And this `LDFLAGS` will be overwritten by the below one, right?

Reply:
forget to delete below, `hack/version.sh` generate LDFLAGS just the same as below, we refactor it to a function because `hack/docker.sh` also need this variable.

Reply:
fixed.